hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/dynamicconfig/WanReplicationConfigTransformer.java

```diff
@@ -271,14 +270,28 @@ DiscoveryConfig toConfig(@Nonnull DiscoveryConfigHolder holder) {
         return discoveryConfig;
     }
 
+    @Nonnull
+    WanSyncConfig toConfig(@Nonnull WanSyncConfigHolder holder) {
+        ConsistencyCheckStrategy strategy = holder.getConsistencyCheckStrategy() == 1
```

Comment:
Since there is no default ConsistencyCheckStrategy defined in the WanSyncConfig class, I think we can use 
```
ConsistencyCheckStrategy strategy = ConsistencyCheckStrategy.getById(holder.getConsistencyCheckStrategy()); 
```

instead and it may be better due to not having to update here if a new ConsistencyCheckStrategy is added

Reply:
Sure, will make the change -- I mainly went by activity on these files; no options added here for 5+ years.